tests/unit/components/form/HdGoogleAutocomplete.spec.js

```diff
@@ -67,4 +67,61 @@ describe('HdGoogleAutocomplete', () => {
 
     expect(wrapper.find('input').attributes().disabled).toBe('disabled');
   });
+
+
+  describe('Supports custom validators with error message', () => {
+    const atLeastTenCharsErrorMessage = 'should be at least ten characters long';
+    const atLeastTenChars = value => ({ isValid: value.length >= 10, errorMessage: atLeastTenCharsErrorMessage });
+
+    const startsWithSpaceErrorMessage = 'should start with empty space';
+    const startsWithSpace = value => ({ isValid: value[0] === ' ', errorMessage: startsWithSpaceErrorMessage });
+
+    test('Simple case', () => {
+      // invalid
+      wrapper.setProps({
+        customValidators: [atLeastTenChars],
+        value: 'too short',
+      });
+
+      wrapper.vm.validate();
+
+      expect(wrapper.vm.isValid).toBe(false);
+      expect(wrapper.vm.error).toBe(atLeastTenCharsErrorMessage);
+
+      // valid
+      wrapper.setProps({
+        customValidators: [atLeastTenChars],
+        value: 'this one is long enough',
+      });
```

Comment:
It is not necessary to set `customValidators` again here, right?

Reply:
@viniciuskneves Technically not, you are right :)

I left it because I thought it makes the test setup more explicit and clear. "It validates these rules against this value". I'm ok with removing it as well